frontend/mocks/db.json

```diff
@@ -1,7 +1,20 @@
 {
   "profile": {
+    "id": 1,
     "name": "Carlo Gino Catapang",
     "alias": "Pogi",
-    "age": 3
+    "age": 3,
+    "profile": "../assets/img/profile.png",
```

Comment:
change this to image

Reply:
anu po ung ibig sabihin nyu sir d2? ung profile property path sir?  babaguhin ko po ung img to image